examples/pubsub-simple/subscriber.py

```diff
@@ -54,4 +60,4 @@ def mytopic_wildcard(event: v1.Event) -> TopicEventResponse:
     return TopicEventResponse('success')
 
 
-app.run(50051)
+app.run(50071)
```

Comment:
```suggestion
app.run(50051)
```

It needs to be port 50051 because the corresponding test README file has an instruction to start Dapr with this app port.